src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2100,14 +2112,15 @@ private void InternalSaveAs(string path, SaveContext saveContext, bool isBackup
                 {
                     AddToRecentFiles(path);
 
-                    if ((currentWorkspaceViewModel?.IsHomeSpace ?? true) && HomeSpace.HasRunWithoutCrash && Model.CurrentWorkspace.IsValidForFDX && currentWorkspaceViewModel.Checksum != string.Empty) 
+                    if ((currentWorkspaceViewModel?.IsHomeSpace ?? true) && HomeSpace.HasRunWithoutCrash && Model.CurrentWorkspace.IsValidForFDX && MLDataIngestionPipelineTest && currentWorkspaceViewModel.Checksum != string.Empty)
```

Comment:
@QilongTang Adding the flag check before so as to avoid the checksum computation as well.

Reply:
Thanks！